internal/chain/beacon/node_test.go

```diff
@@ -133,7 +133,10 @@ type BeaconTest struct {
 	scheme   *crypto.Scheme
 }
 
-func NewBeaconTest(ctx context.Context, t *testing.T, clock clock.FakeClock, n, thr int, period time.Duration, genesisTime int64, beaconID string) *BeaconTest {
+func NewBeaconTest(
+	ctx context.Context, t *testing.T, c clock.FakeClock,
```

Comment:
personally I hate this formatting, but it seems the formatter doesn't complain :p

Reply:
Better than having too long lines